File: C/URI/1073.c
#include <stdio.h>
 
int main() {
 
  int numero, pot, n;
        scanf("%d", &numero);
        n = 2; 
        while( numero > 1)
        {
                pot = n * n;
                printf("%d^2 = %d\n", n, pot);
                n = n + 2;
                numero = numero -2;
        }

 
    return 0;
}

File: JS/URI/1073.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');


var pot;
var n = 2;
var numero = parseInt(lines.shift());
while (numero > 1) {
    pot = n * n;
    // printf("%d^2 = %d\n", n, pot);
    // cout << n << "^2 = " << pot << "\n";
   console.log(n + "^2 = " + pot);
    n = n + 2;
    numero = numero - 2;
}
